### PR TITLE
Update configure_docker_daemon.md

### PR DESCRIPTION
**Pull request by alexandair**
Fix typos.

### CODE REVIEW

File: virtualization/windowscontainers/docker/configure_docker_daemon.md [hunk 1]
```diff
@@ -31,19 +31,19 @@ Download the Docker Engine
 
 The latest version may always be found at https://master.dockerproject.org . This sample uses the latest from the v1.13-development branch. 
 
-```none
+```powershell
 Invoke-WebRequest "https://download.docker.com/components/engine/windows-server/cs-1.12/docker.zip" -OutFile "$env:TEMP\docker.zip" -UseBasicParsing
```
Comment (alexandair) [inline virtualization/windowscontainers/docker/configure_docker_daemon.md]: 1.13.0-dev build is mentioned in text, but a command downloads a cs-1.12 build.

Comment (PatrickLang) [inline virtualization/windowscontainers/docker/configure_docker_daemon.md]: I need to look at this a bit closer for consistency. The Windows 10 documents are supposed to use v1.13-dev until v1.13 is final. Windows Server 2016 is supposed to use cs-1.12.2 which is a preview of the commercially supported Docker Engine.

Comment (PatrickLang) [inline virtualization/windowscontainers/docker/configure_docker_daemon.md]: @enderb-ms is probably the best person to review this.

Comment (alexandair) [inline virtualization/windowscontainers/docker/configure_docker_daemon.md]: I've had v1.13-dev on my Windows 10, but needed to switch to cs-1.12.2 after GA version of container images for Nano Server where uploaded to Docker Hub. What I have on Windows 10 now is the following:
```
PS C:\> docker version
Client:
 Version:      1.12.2-cs2-ws-beta
 API version:  1.25
 Go version:   go1.7.1
 Git commit:   050b611
 Built:        Tue Oct 11 02:35:40 2016
 OS/Arch:      windows/amd64

Server:
 Version:      1.12.2-cs2-ws-beta
 API version:  1.25
 Go version:   go1.7.1
 Git commit:   050b611
 Built:        Tue Oct 11 02:35:40 2016
 OS/Arch:      windows/amd64
```

Comment (alexandair) [inline virtualization/windowscontainers/docker/configure_docker_daemon.md]: I've had v1.13-dev on my Windows 10, but needed to switch to cs-1.12.2 after GA version of container images for Nano Server where uploaded to Docker Hub. What I have on Windows 10 now is the following:
```
PS C:\> docker version
Client:
 Version:      1.12.2-cs2-ws-beta
 API version:  1.25
 Go version:   go1.7.1
 Git commit:   050b611
 Built:        Tue Oct 11 02:35:40 2016
 OS/Arch:      windows/amd64

Server:
 Version:      1.12.2-cs2-ws-beta
 API version:  1.25
 Go version:   go1.7.1
 Git commit:   050b611
 Built:        Tue Oct 11 02:35:40 2016
 OS/Arch:      windows/amd64
```

Comment (PatrickLang) [inline virtualization/windowscontainers/docker/configure_docker_daemon.md]: The cs builds are intended for Windows Server 2016 where Docker commercial support is available. 

At this point, [v1.13 is in code freeze](https://github.com/docker/docker/wiki) and should be working on Windows 10. If it's failing, please be sure to get an issue filed at https://github.com/docker/docker/issues

**Comment by PatrickLang** [comment]
Changes look good! I'll fix the v1.13 issue separately

**Pull request merged**